core/rec-x86/rec_x86_asm.cpp

```diff
@@ -89,7 +91,11 @@ naked void ngen_mainloop(void* cntx)
 		jnz do_iter;
 		ret;
 
-do_iter:
+	do_iter:
```

Comment:
It would be faster here to use the `return-to-fragment-via-rv` approach to avoid paying the cost of the if in the main loop. I guess that's good enough for now, and on x86 we can eat the perf hit anyway